### DISCUSSION TITLE
Authentication SAML with Loopback 4

### DISCUSSION THREAD

**Discussion by ultrex** [loopbackio/loopback-next#8060] [Q&A]
Does someone have an example of how to configure authentication SAML into loopback 4?

I have several days looking for information about it, but I have not found a full example.

The official documentation just has examples with connections with social networks like Facebook, Google, etc, y other types but not with SAML.

Thank you.

**Comment by juanpaie** [comment]
I'm also looking for a SAML/LB4 integration tutorial. I have only found examples with social networks (https://github.com/loopbackio/loopback-next/tree/master/examples/passport-login)
